devfiles/quarkus/devfile.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: quarkus-
+projects:
+  -
+    name: quarkus-quickstarts
+    source:
+      type: git
+      location: "https://github.com/quarkusio/quarkus-quickstarts.git"
+      sparseCheckoutDir: /getting-started/
+components:
+  -
+    type: chePlugin
+    id: redhat/quarkus/latest
+  -
+    type: dockerimage
+    alias: maven
+    image: quay.io/eclipse/che-java8-maven:nightly
```

Comment:
we should include the ability to run the native compilation to this devfile

Reply:
could you please clarify why do we need that?

Reply:
With Quarkus, you have a way to produce a tiny native executable https://quarkus.io/guides/building-native-image

for instance:
https://github.com/sunix/che-quarkus-demo/blob/8f4f4fa8e09158853e9cd900c0fe93cac18da484/devfile.yaml#L109-L121

Reply:
looks like GraalVM Native Image generation requires at least 2Gi RAM to perform a native image build. So, if we give this value + 1,5Gi for Quarkus VS Code plugin (Java LS, Quarkus LS, Java Debugger) container, it'd be more than 3Gi - it is too much for che.openshift.io

Reply:
To me we have to produce a devfile for che. Not che.openshift.io.

Reply:
upstream devfile registry is used on Hosted Che, so all the devfiles in the registry must be compatible with Hosted Che. Otherwise, we need to come up with Hosted Che specific devfile registry (smth. that should be avoided)

Reply:
> looks like GraalVM Native Image generation requires at least 2Gi RAM to perform a native image build. So, if we give this value + 1,5Gi for Quarkus VS Code plugin (Java LS, Quarkus LS, Java Debugger) container, it'd be more than 3Gi - it is too much for che.openshift.io

Could you give a try in a single container with vscode ext for java/quarkus and running on a container with graalvm ?